front/style/pages/_donation.scss

```diff
@@ -389,6 +389,18 @@
         &__city {
             width: 208px;
         }
+        .form__row--stretch {
+            margin-bottom: 15px;
```

Comment:
You can use directly the `b__nudge--bottom-medium` class in your template for this margin bottom

Reply:
fixed